src/Weather/Rasp/RaspStyle.cpp

```diff
@@ -116,8 +116,312 @@ static constexpr ColorRamp rasp_colors[6][NUM_COLOR_RAMP_LEVELS] = {
     { 300, { 255, 153, 0 }},
     {1000, { 255, 102, 0 }},
   },
+
+  // Thermalmap.info Color schemes
+
+  // Vertical speed (wave, Convergence)
+  // Offset 5000 cm/s
+  // Multiplyfactor 1000
+  // Blue to white to Purple
```

Comment:
I have no idea what this multi-line comment expressed in single-line comment syntax means

Reply:
Please see our style guide for comment formatting: 
https://xcsoar.readthedocs.io/en/latest/policy.html?highlight=comment#code-style

Reply:
I reformated them, but i'm also unsure about what exactly the lines mean. @Developer66 care to comment?